src/Api/Controllers/OrganizationExportController.cs

```diff
@@ -12,38 +13,53 @@ namespace Bit.Api.Controllers;
 [Authorize("Application")]
 public class OrganizationExportController : Controller
 {
+    private readonly ICurrentContext _currentContext;
     private readonly IUserService _userService;
     private readonly ICollectionService _collectionService;
     private readonly ICipherService _cipherService;
     private readonly GlobalSettings _globalSettings;
 
     public OrganizationExportController(
+        ICurrentContext currentContext,
         ICipherService cipherService,
         ICollectionService collectionService,
         IUserService userService,
         GlobalSettings globalSettings)
     {
+        _currentContext = currentContext;
         _cipherService = cipherService;
         _collectionService = collectionService;
         _userService = userService;
         _globalSettings = globalSettings;
     }
 
     [HttpGet("export")]
-    public async Task<OrganizationExportResponseModel> Export(Guid organizationId)
+    public async Task<IActionResult> Export(Guid organizationId)
     {
         var userId = _userService.GetProperUserId(User).Value;
 
         IEnumerable<Collection> orgCollections = await _collectionService.GetOrganizationCollections(organizationId);
         (IEnumerable<CipherOrganizationDetails> orgCiphers, Dictionary<Guid, IGrouping<Guid, CollectionCipher>> collectionCiphersGroupDict) = await _cipherService.GetOrganizationCiphers(userId, organizationId);
 
-        var result = new OrganizationExportResponseModel
+        // Backward compatibility with versions 2022.9.0 and 2022.10.0 that use ListResponseModel
+        if (new[] { "2022.9.0", "2022.10.0" }.Contains(_currentContext.ClientVersion))
```

Comment:
Would be nice to have a parser for version and just mark it as `_currentContext.ClientVersion.OldeOrEqualTo("2022.10.0")`. But that seems somewhat out of  scope

Reply:
Ideally, depends when QA can test it.